### PR TITLE
Explain how to put the sid File into RFC 9195 YANG Instance Data

### PR DESCRIPTION
**Pull request by cabo**

### CODE REVIEW

File: draft-ietf-core-sid.md [hunk 1]
```diff
@@ -738,6 +738,37 @@ The following Activity diagram summarizes the update of a YANG module and its as
...
+    "content-schema": {
+      "module": "ietf-sid-file@2021-11-16"
+    },
+    "content-data": "<replace by SID file contents>"
+  }
+}
+~~~
+
+(Note that the content-data are not actually embedded as a string, but
+as the JSON value of the SID file.  )
```
Comment (mcr) [inline draft-ietf-core-sid.md:766]: So, it's actually the JSON dict/map, { "foo" : "bar"}, not "{ \"foo\":\"bar\" }", right?
Why not put that into the example then?

Comment (abierman) [reply draft-ietf-core-sid.md:766]: It seems like "content-data" should be an object, not a string.
This seems like a significant change to the document.

     "content-data": {
          <replace by SID file contents>
      }

**Comment by cabo** [comment]
Yeah, I tried to make the code block a valid piece of JSON and may have obscured what is going on more than what I could compensate with that paragraph.  Will update.

File: draft-ietf-core-sid.md (hunk 1 above)
Comment (cabo) [reply draft-ietf-core-sid.md:766]: Yeah, I tried to make the code block a valid piece of JSON and may have obscured what is going on more than what I could compensate with that paragraph. Will update.

**Pull request merged**